client/pages/home.tsx

```diff
@@ -11,6 +11,8 @@ import { withApollo } from '../utils/withApollo';
 
 const Container = styled.div`
   font-family: Montserrat;
+  margin: 0 auto;
+  max-width: 1500px;
```

Comment:
I think if the goal is to limit content width, we should add it to the <main> tag on App.js or to the whole container in App.js, that way we don't have to explicitly set it in every page